crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1430,6 +1564,36 @@ impl TryFrom<StoredObject> for Coin {
     }
 }
 
+impl TryFrom<SuiStake> for Stake {
```

Comment:
I think the TryFrom is unnecessary as there doesn't appear to be anything that could fail